src/core/services/layout/layout.scss

```diff
@@ -510,3 +511,10 @@
   }
 
 }
+
+// IE mediaQuery hack for 8,9,10 to set the flex-basis properly for 'flex' values
```

Comment:
More background on this? What it's fixing and a reference link for the fix would be good.